MBBSEmu/Module/ModuleConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace MBBSEmu.Module
+{
+    /// <summary>
+    ///     Module Configuration
+    /// </summary>
+        
+    public class ModuleConfiguration
+    {
+        /// <summary>
+        ///     Module "Identifier" from moduleConfig.json or commandline
+        /// </summary>
+        public string ModIdentifier { get; set; }
```

Comment:
I would rather these be `ModuleIdentifier` or just `Identifier`, as the implied context is it's already in relation to a module.

Reply:
Changed. Love "Refactor->Rename" :)